examples/server/server.cpp

```diff
@@ -3649,20 +3649,30 @@ int main(int argc, char ** argv) {
             oaicompat = true;
             prompt = body.at("input");
         } else if (body.count("content") != 0) {
-            // with "content", we only support single prompt
-            prompt = std::vector<std::string>{body.at("content")};
+            prompt = body.at("content");
         } else {
             res_error(res, format_error_response("\"input\" or \"content\" must be provided", ERROR_TYPE_INVALID_REQUEST));
             return;
         }
 
+        // with "content", we only support single prompt
+        if (!oaicompat && prompt.type() != json::value_t::string) {
+            res_error(res, format_error_response("\"content\" must be a string", ERROR_TYPE_INVALID_REQUEST));
+            return;
+        }
+
```

Comment:
I wonder what is the reason to apply this restriction on the `"content"` field? Why don't we treat it as an alias to `"input"` and allow the same inputs for it?